setup.py

```diff
@@ -1,9 +1,9 @@
 from distutils.core import setup, Extension
 from pip.req import parse_requirements
-install_reqs = parse_requirements('requirements.txt', session=False)
+# install_reqs = parse_requirements('requirements.txt', session=False)
```

Comment:
Did you mean to leave these commented out? It would be nice to have explanations above commented out code.